deployments/myhub/hubploy.yaml

```diff
@@ -11,8 +11,8 @@ images:
       service_key: gcr-key.json
     aws:
       # Pushes to Amazon ECR
-      project: # TODO: AWS account id
-      zone: # TODO: Zone in which your container image should live. Match your cluster's zone
+      account_id: # TODO: AWS account id
+      region: # TODO: Zone in which your container image should live. Match your cluster's zone
```

Comment:
The comment still reference "Zone", same for the lines below. I'm not familiar with AWS so I'll let you update this without a guess-suggestion from me.

Reply:
Ah, they are supposed to be `region`. Forgot to check the comments. Thanks!